REFERENCE.md

```diff
@@ -10,11 +10,11 @@ _Public Classes_
 * [`sensu`](#sensu): Base Sensu class
 * [`sensu::agent`](#sensuagent): Manage Sensu agent
 * [`sensu::backend`](#sensubackend): Manage Sensu backend
-* [`sensu::backend::datastore::postgresql`](#sensubackenddatastorepostgresql): 
```

Comment:
REFERENCE.md is created during a release and should not be part of the PR.

Reply:
Fixed, had generated to verify the change and accidentally committed.